snuba/query/processors/mandatory_condition_applier.py

```diff
@@ -0,0 +1,22 @@
+from snuba.query.logical import Query
+from snuba.clickhouse.processors import QueryProcessor
+from snuba.request.request_settings import RequestSettings
+
+# from snuba.request import Request
+# from snuba.datasets.dataset import Dataset
+from snuba.query.conditions import combine_and_conditions
+
+
+class MandatoryConditionApplier(QueryProcessor):
```

Comment:
Could you please add a docstring to this class explaining briefly what this processor does ?
A good approach to provide this comment is focusing on. giving context, explaining what its responsibilities are and why more than giving details on every single line.

I think a successful way in general is to answer the following questions:
1) Provide a 2 line maximum description of the responsibility of this processor
2) How your class is supposed to be used and in which circumstances. In this case that is implicit in the fact this is a query processor. So you should skip this.
3) How this fit in a larger context (again in this case this is implied by the class being a query processor, so the way it is supposed to be. used is defined by the parent class, no need to answer this)
4) Limitations if any. There aren't in this case.

So a 2 lines explanation for this specific processor is engouh.